src/index.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import App from './components/App/App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
   </React.StrictMode>
```

Comment:
`Browser.Router` should wrap the strict mode. I remember the instructor said something about this and it's the way the lesson was formulated. It has something to do with how you want `React.StrctMode` to wrap app... but I'm not exactly sure why.

Reply:
Gonna keep this as is per our findings from the React Router example projects!